site/content/en/docs/tasks/run/leaderworkerset.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Run LeaderWorkerSet"
+linkTitle: "LeaderWorkerSet"
+date: 2025-02-17
+weight: 6
+description: >
+   Run a LeaderWorkerSet as a Kueue-managed workload for each group.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running LeaderWorkerSet.
+
+We demonstrate how to support scheduling LeaderWorkerSet in Kueue based on the
+[Plain Pod Group](https://kueue.sigs.k8s.io/docs/tasks/run/plain_pods/) integration,
+where LeaderWorkerSet is represented as a separate, independent workload for each
+group.
+
+This guide is for [serving users](/docs/tasks#serving-user) that have a basic understanding of Kueue.
```

Comment:
We should probably add it here to the list for serving workloads: https://kueue.sigs.k8s.io/docs/tasks/.

Reply:
Done